### PR TITLE
Adjusted *SEM and SEMEVAL names

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

File: data/yaml/venues.yaml [hunk 1]
```diff
@@ -732,7 +728,7 @@ semdeep:
   acronym: SemDeep
   name: Workshop on Semantic Deep Learning
 semeval:
-  acronym: '*SEMEVAL'
+  acronym: SEMEVAL
```
Comment (nschneid) [inline data/yaml/venues.yaml:731]: acronym: SemEval
name: International Workshop on Semantic Evaluation

**Comment by nschneid** [comment]
In 2012 & 2013 *SEM and SemEval were part of a joint event, in two volumes with old-style letter `S`. How will this work?

**Comment by mjpost** [comment]
That event is listed here: https://www.aclweb.org/anthology/venues/semeval/

If you want to cross-list this and other old events under https://www.aclweb.org/anthology/venues/starsem, they have to be added to `joint.yaml`.

**Comment by nschneid** [comment]
Ah, so 2012 is actually just one volume, and it can be joint.

I'm mainly wondering about 2013: technically SemEval is just volume 2, so what to do about volume 1? Can we say volume 1 is like an ordinary main conference (so *SEM as its only venue) and volume 2 is a workshop at a conference (so *SEM, SemEval, and WS as venues)? With that strategy is it a problem that the volume 1 IDs are also `S`?

File: data/yaml/joint.yaml [hunk 2]
```diff
@@ -1053,6 +1053,8 @@ ssst:
   - W14-40
   2015:
   - W15-10
+starsem:
+  2020: [S12-1, S13-1, S14-1, S15-1, S16-2, S17-1, S18-2, S19-1]
```
Comment (nschneid) [inline data/yaml/joint.yaml:1057]: not sure why "2020"—should these be divided by year?

Comment (mjpost) [reply data/yaml/joint.yaml]: Yes, careless.

**Comment by mjpost** [comment]
@nschneid can you approve? *SEM / SemEval folks would like this live before tomorrow.

**Comment by akoehn** [comment]
@nschneid that would be indeed helpful!  I always feel like for every interesting piece of the anthology, there are currently only 1-2 people completely understanding it.  Those yamls are a thing where I am also not quite sure.

(NB: I think that our code & logic is generally well structured and clean, there are just always some tricky intricacies)

**Comment by davidweichiang** [comment]
I've previously put some documentation into this repo's wiki. I think that would be a good place for this, too.

**Comment by mjpost** [comment]
@davidweichiang where on the wiki? I couldn't find anything. I'll dump what I know today.

**Comment by nschneid** [comment]
@mjpost did you want to merge?

**Comment by mjpost** [comment]
I created a page, see if it helps: https://github.com/acl-org/acl-anthology/wiki/Venues,-Volumes,-and-Events

**Comment by nschneid** [comment]
Thanks @mjpost, this is great. A catch is that for citation purposes there are *three* types of joint events that we need to be able to distinguish (e.g. for Zotero import):

1. Two venues with joint proceedings, e.g. ACL-IJCNLP
2. A conference colocated with a bigger conference, e.g. CoNLL colocated with EMNLP
3. A workshop colocated with a main conference, e.g. DMR colocated with COLING

In the past we were able to distinguish the 2nd and 3rd cases because all workshop events were listed under the WS venue as well as the workshop venue and the main conference venue. Should we continue doing that going forward? Or should we mark at the venue level whether it is a workshop, and expose that on the event/volume/paper pages? I suppose `is_toplevel` expresses something similar (though we could quibble about WMT and IWCS no longer being workshops); however AFAIK its value is not available on paper pages.

**Comment by mbollmann** [comment]
I made a new issue with some thoughts about this: #1164

**Comment by mbollmann** [comment]
But to address point number 3 here, I think it is a mistake that e.g. DMR is not associated with the "WS" venue, which is probably a result of the system being unintuitive so that it's easy to forget to do this.

**Pull request merged**